docs/platform-integration/essentials/get-started.md

```diff
@@ -1,95 +1,84 @@
 ---
-title: "Get Started with Xamarin.Essentials"
-description: "Get started with Xamarin.Essentials, which provides a single cross-platform API that works with any iOS, Android, or UWP application."
-ms.date: 05/11/2020
+title: "Get Started with Essentials"
+description: "Get started with Essentials in .NET MAUI. Essentials provides a single cross-platform API that your app can use to access device features, operating system functionality, and phone support, among other capabilities."
+ms.date: 08/11/2021
 no-loc: ["Microsoft.Maui", "Microsoft.Maui.Essentials"]
 ---
 
-# Get Started with Xamarin.Essentials
+# Get started with .NET MAUI Essentials
 
-Xamarin.Essentials provides a single cross-platform API that works with any iOS, Android, or UWP application that can be accessed from shared code no matter how the user interface is created. See the [platform & feature support guide](platform-feature-support.md) for more information on supported operating systems.
+.NET Multi-platform App UI (.NET MAUI) Essentials provides a single cross-platform API that works with any iOS, Android, or UWP application. Essentials is accessed from cross-platform friendly code that is ignorant of the platform it's run on. Some APIs do require platform-specific configure or setup, but that's the exception rather than the rule. For more information about platform and operating system support, see [Platform Support](platform-feature-support.md).
 
-## Installation
+## Migrating from Xamarin.Forms
 
-Xamarin.Essentials is available as a NuGet package and is included in every new project in Visual Studio. It can also be added to any existing projects using Visual Studio with the following steps.
+Unlike Xamarin.Forms Essentials, .NET MAUI Essentials is included with .NET MAUI. You're no longer required to install a NuGet package or add reference to the Essentials library.
 
-1. Download and install [Visual Studio](https://visualstudio.microsoft.com/) with the [Visual Studio tools for Xamarin](~/xamarin-forms/get-started/installation/index.md).
+## Setup
```

Comment:
I'd put a todo comment here to revisit this. For starters the min requirement for MAUI on Android is now API level 21. I *think* the target Android version might be API 30 now, too.

Also, the stuff about the Android support libraries is now wrong. It'll require the AndroidX support libraries.

Basically, it's out of date.